lib/avtp_pipeline/platform/Linux/openavb_time_osal.c

```diff
@@ -71,7 +71,7 @@ static bool x_timeInit(void) {
 		return FALSE;

```

Comment:
gptpinit error reporting sense has changed; the code here needs to be changed to check for < 0 rather than 0.